test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/TestServer.cs

```diff
@@ -65,8 +65,11 @@ public TestServer(RequestDelegate app, TestServiceContext context, ListenOptions
             }
             catch
             {
-                _transport.UnbindAsync().Wait();
-                _transport.StopAsync().Wait();
+                if (_transport != null)
```

Comment:
Yes, it was throwing. The NRE thrown from the catch was hiding the underlying issue so I didn't figure out what was going on.